core/src/rustic_ai/core/guild/dsl.py

```diff
@@ -195,6 +195,7 @@ class GuildSpec(BaseModel):
         agents (list[AgentSpec]): A list of agents in the guild.
         dependency_map (Dict[str, DependencySpec]): A mapping for guild's dependency to resolver class.
         routes (RoutingSlip): The routes to be attached to every message coming in the guild.
+        status (str): The status of the guild, e.g., "running", "stopped", "archived".
```

Comment:
It would be useful to refer to the enum in the docs so ppl can check the allowed values